get_map_snp.sh

```diff
@@ -36,4 +36,11 @@ while IFS=':' read -r CHR REGION; do
     unzip -p ${REF_MAP}${REF_GEN}.map.zip plink.${CHR}.${REF_GEN}.map | \
         awk -v OFS='\t' -F' ' '{ print $1, $3, $4 }' \
         >  ${REF_MAP}/${REF_GEN}_${CHR_NUM}.map
```

Comment:
Could you also update the old map ?
```suggestion
        awk -v OFS='\t' -F' ' '{
            if ($1 !~ /^chr/) $1 = "chr" $1
            print $1, $3, $4
        }' \
        >  ${REF_MAP}/${REF_GEN}_${CHR}.glimpse.map
```